01-node-tutorial/answers/writeWithPromisesThen.js

```diff
@@ -0,0 +1,27 @@
+
+
+const { writeFile, readFile, appendFile } = require("fs").promises;  
+
+const lines =[
+    "Line One \n",
+    "Line Two \n",
+    "Line Tree \n"
+]
+
+const filename = "./temporary/temp.txt"
+
+// Async function to write three lines to temp.txt
+const writer = async () => {
+    writeFile(filename, lines[0])
+    .then(() => {  
+    return appendFile(filename, lines[1]) 
+    })  
+    .then(() => {  
+        return appendFile(filename, lines[2]) 
+    }) 
+    .catch((error) => {  
+        console.log("An error occurred: ", error)  
+    })  
+}
```

Comment:
The indentation for this function is inconsistent.
```suggestion
const writer = async () => {
    writeFile(filename, lines[0])
        .then(() => {  
            return appendFile(filename, lines[1]) 
        })  
        .then(() => {  
            return appendFile(filename, lines[2]) 
        }) 
        .catch((error) => {  
            console.log("An error occurred: ", error)  
        })  
}
```